P/PCRE2/bundled/patches/sljit-nomprotect.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/sljit_src/sljitExecAllocator.c b/sljit_src/sljitExecAllocator.c
+index 2e1c138..bae8cd6 100644
+--- a/sljit_src/sljitExecAllocator.c
++++ b/sljit_src/sljitExecAllocator.c
+@@ -182,10 +182,12 @@ static SLJIT_INLINE void* alloc_chunk(sljit_uw size)
```

Comment:
The file seems to be in `src/sljit/sljitExecAllocator.c`, at a different line and with a different context

Reply:
Ok, the different context is due to the other patch we carry, so that's ok.  And you also fix the path on the command line :thinking:

Reply:
Right; it's because it's coming from the `sljit` repo, not the PCRE repo.

Reply:
Wrong path of the patch file, not the patched one :sweat_smile: